### ISSUE TITLE
Error when adding control crossref options in defaults.yaml file

### ISSUE THREAD

**Issue by rriemann** [lierdakil/pandoc-crossref#259]
Hello,

I use pandoc with a defaults file: `pandoc -d my-file.yaml`

In this file, I would like to put some options for `pandoc-crossref`:

```yaml
metadata:
  cref: true
  linkReferences: true
  nameInLink: true
  bibliography: ./bibliography.bib
```

However, an error prevents doing so:

```
WARNING: pandoc-crossref was compiled with pandoc 2.9.2 but is being run through 2.9.2.1. This is not supported. Strange things may (and likely will) happen silently.
pandoc-crossref: Expected bool in metadata field cref but got string
CallStack (from HasCallStack):
  error, called at lib/Text/Pandoc/CrossRef/Util/Meta.hs:73:25 in pandoc-crossref-0.3.6.2-inplace:Text.Pandoc.CrossRef.Util.Meta
Error running filter pandoc-crossref:
Filter returned error status 1
```

**Comment by lierdakil** [comment]
Apparently that's a bug in pandoc. Seems like it parses bools in defaults.yaml incorrectly (i.e. as strings). Let me know if you want me to raise an issue on the pandoc repo or if you would prefer to do that yourself. As a temporary measure, I can suggest using `pandoc-crossref.yaml` instead.

**Comment by lierdakil** [comment]
NVM, I'll make a PR

**Comment by lierdakil** [comment]
I believe the fix is released in pandoc 2.10.
